styles/component-style/badge.css

```diff
@@ -0,0 +1,87 @@
+* {
+    --background-color-one:blue;
+    --background-color-two:brown;
+    --background-color-three:green;
+    --background-color-four:red;
+}
+
+
+.main-container {
+    display:flex;
+    flex-direction:column;
+}
+
+.badge-icon {
+    display: flex;
+    column-gap: 3rem;
+    padding-top:1.5rem;
+    padding-left:1.5rem;
+    background-color: var(--background-color);
+    width:50vw;
+}
+
+.badge-icon img {
+    height:2rem;
+    width:2rem;
+}
+
+.badge {
+    color:white;
+    border-radius: 50%;
+    height:1rem;
+    width:1rem;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    background-color: black;
+}
+
+p2 {
+    transform: translate(130%,-250%);
+}
+
+.badge-icon-one{
+ background-color: var(--background-color-one);
+}
+
+.badge-icon-two {
+  background-color: var(--background-color-two);  
+}
+
+.badge-icon-three {
+  background-color: var(--background-color-three);  
+}
+
+.badge-avatar img {
+    border-radius:50%;
+    height:4rem;
+    width:4rem;
+}
+
+.badge-avatar {
+    display: flex;
+    column-gap: 3rem;
```

Comment:
Suggestion : You are using flex so often. So it will be better to create separate utility classes for them.